docs/release-notes/8851-force-default-passwords.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**New Features**
+
+-  Add a configuration setting, ``initial_user_password``, to the master configuration file forcing the setup of an initial user password for the built-in ``determined`` and ``admin`` users during the first launch, specifically when a cluster's database is bootstrapped.
```

Comment:
```suggestion
-  Add a configuration setting, ``initial_user_password``, to the master configuration file forcing the setup of an initial user password for the built-in ``determined`` and ``admin`` users during the first launch, specifically when a cluster's database is bootstrapped. Users with existing clusters should ensure they have passwords set for all users if those clusters are publicly accessible.
```

Reply:
added the changes https://github.com/determined-ai/determined/pull/8864/commits/e193924793ea6c32de91ddacf4b575cef34eefc3